build/storage/scripts/build_container.sh

```diff
@@ -13,22 +13,30 @@ root_dir="${script_dir}/.."
 source "${script_dir}"/spdk_version.sh
 "${script_dir}"/prepare_to_build.sh
 
+possible_containers=("storage-target" "ipu-storage-container" \
+    "host-target" "traffic-generator" "test-driver" "ipdk-unit-tests" \
+    "cmd-sender")
+container_to_build="${1}"
```

Comment:
Don't we check if the container to build is one from the allow list?

Reply:
Yes, we do check, but in another place on line 50. I will move the definitions back closer to that check

Reply:
Yes, we do check, but in another place on line 50. I will move the definitions back closer to that check